safe/gisv4/vector/summary_3_analysis.py

```diff
@@ -132,16 +131,16 @@ def analysis_summary(aggregate_hazard, analysis, callback=None):
         value = hazard_count_field['field_name'] % column
         analysis.keywords['inasafe_fields'][key] = value
 
-    field = create_field_from_definition(affected_count_field)
-    analysis.addAttribute(field)
-    analysis.keywords['inasafe_fields'][affected_count_field['key']] = (
-        affected_count_field['field_name'])
-
     field = create_field_from_definition(total_affected_field)
     analysis.addAttribute(field)
     analysis.keywords['inasafe_fields'][total_affected_field['key']] = (
         total_affected_field['field_name'])
 
+    field = create_field_from_definition(total_field)
```

Comment:
Ok, I think it is clear for me now, that Total field is the total count (from exposure), and total_affected_field is the count of exposure considered as affected.